MoveCardAfterReview/ReadMe.md

```diff
@@ -37,6 +37,7 @@ jobs:
     # PRs from forks don't have required token authorization
     if: |
         github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.review.author_association != 'NONE'
```

Comment:
Will this be true if the contributor is part of an organization other than SonarSource? Should the check be something like:

```
github.event.review.author_association != 'SonarSource'
```

Or whatever the id for SonarSource is.

Reply:
The values here can be COLLABORATOR, CONTRIBUTOR, FIRST_TIMER, FIRST_TIME_CONTRIBUTOR, MANNEQUIN, MEMBER, NONE, OWNER
I played safe and excluded the one that has nothing to do with the repo, as there's a long list of weird values that we want to move on with.

Reply:
Thanks for explaining!